core/templates/dev/head/pages/skill-editor-page/services/skill-editor-state.service.spec.ts

```diff
@@ -65,10 +65,10 @@ describe('Skill editor state service', function() {
             skill: self.newBackendSkillObject,
             groupedSkillSummaries: {
               Name: [{
-                id: 'skill_id_1',
+                id: '1',
```

Comment:
Any reason to change this? I thought the previous version was more readable.

Reply:
This was to make the `id` field contain the skill ID of the skill used elsewhere in the test, and that skill (`self.newBackendSkillObject`) had the ID 1.

Reply:
Could that skill ID be changed to skill_id_1 throughout? "1" is too easy to confuse with a number/index of something.

Reply:
Done